assets/pages/cfp/index.jinja

```diff
@@ -88,11 +88,11 @@
             </div>            
         </div>
         
-        <div class="card">
-            <div class="title">
+        <div class="card blue-theme">
```

Comment:
Please refers to https://github.com/404busters/hkoscon-2017/blob/master/assets/pages/about/coc.jinja